src/pages/permissions/panels/EditGroupPermissionsForm.tsx

```diff
@@ -38,6 +40,23 @@ const EditGroupPermissionsForm: FC<Props> = ({
 }) => {
   const notify = useNotify();
   const [search, setSearch] = useState("");
+  const { canViewPermissions } = useServerEntitlements();
+  const { canEditGroup } = useGroupEntitlements();
+  const getPermissionEditRestriction = () => {
```

Comment:
```suggestion
  const getEditRestriction = () => {
```
Maybe a more consistent naming?